po/sv.po

```diff
@@ -3451,7 +3451,7 @@ msgstr "Sök order"
 
 #: gnucash/gnome/dialog-payment.c:237
 msgid "You must enter a valid account name for posting."
-msgstr "Du måste ange ett giltigt kontonamn för registrering."
+msgstr "Du måste ange ett giltigt kontonamn för flytt."
```

Comment:
Alsop this and your following changes seem problematic. They deal with processing invoices, not buy against cash or other direct payment. Perhaps read the [manual](https://gnucash.github.io/gnucash-docs/C/gnucash-manual/chapter_busnss.html) or [guide](https://gnucash.github.io/gnucash-docs/C/gnucash-guide/chapter_bus_features.html)?